src/hooks/useNavLink.js

```diff
@@ -37,7 +37,10 @@ const useNavLink = (navObj) => {
     event.preventDefault()
 
     const elem = document.querySelector(`#${navTargetId}`)
-    if (!isInViewport(elem)) elem.scrollIntoView({ behavior: "smooth" })
+    if (!isInViewport(elem)) {
+      const y = elem.getBoundingClientRect().top + window.pageYOffset - 90 // do not scroll to top, leave space for header row
```

Comment:
Is this appropriate if the title spills onto multiple lines?

Reply:
hmm, excellent question - I'm trying to figure out how to account for that (and to find an example of a label that spans lines)  ... shrinking window size doesn't wrap any headers I can find